data/registry_model/test/test_interface.py

```
@@ -1475,3 +1475,147 @@ def test_lookup_repository_cache_stale_after_delete_without_invalidation(
+    assert ref1.digest in digests_from_db
+    assert ref2.digest in digests_from_db
+
+    # Fetch from cache and verify all fields survived roundtrip
+    cached = oci_model.lookup_cached_referrers_for_manifest(
+        model_cache, repository_ref, subject
+    )
+
+    for orig, from_cache in zip(
+        sorted(referrers, key=lambda m: m.digest),
+        sorted(cached, key=lambda m: m.digest),
+    ):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)